README.md

```diff
@@ -1,23 +1,27 @@
-# Merkle Multiproof Inputs Generation For Forge
+> [!WARNING]
+> Note that this library has not had a security review yet, is not gas-efficient, and should only be used for testing.
 
-This is a simple library to generate Merkle Multiproof inputs for OpenZeppelin's `MerkleProof` library. It is written in Solidity and can be used in Forge framework. The library has been tested to work with at most 10k of arbitrary leaves and arbitrary indices of any size.
+# Merkle MultiProof and SingleProof Inputs Generation For Forge
 
-Note that this library is not gas-efficient and should be used for testing purposes only.
+This simple library generates Merkle MultiProof and SingleProof inputs for OpenZeppelin's `MerkleProof` library. It is written in Solidity and can be used in the Forge framework. The library has been tested to work with at most 10k arbitrary leaves and arbitrary indices of any size.
```

Comment:
I think it'd be a good idea to link the OZ code and also note the version of OZ we're using. Some of the older version of OZ merkle proof wouldn't be compatible.